x-pack/plugins/security_solution/public/timelines/components/side_panel/hooks/use_detail_panel.tsx

```diff
@@ -48,8 +55,13 @@ export const useDetailPanel = ({
   scopeId,
   tabType = TimelineTabs.query,
 }: UseDetailPanelConfig): UseDetailPanelReturn => {
+  const { telemetry } = useKibana().services;
   const { browserFields, selectedPatterns, runtimeMappings } = useSourcererDataView(sourcererScope);
```

Comment:
Flyout throws an error when open in events table -> session view. I wonder if we can use the timeline patterns here?

```const { selectedPatterns } = useTimelineDataFilters(isActiveTimeline(scopeId));```

![image](https://github.com/elastic/kibana/assets/18648970/8ceed5b6-174c-454b-9bd0-ccc8178b4bf4)

per https://github.com/elastic/kibana/issues/130641

Reply:
Yea, this has been a lingering issue per this comment if it helps: https://github.com/elastic/kibana/issues/130641#issuecomment-1108744331

Reply:
yeah I can reproduce the issue on `main`
![Screenshot 2024-04-18 at 4 22 32 PM](https://github.com/elastic/kibana/assets/17276605/dc363a49-a69a-497e-a99c-6cfc15f09ee1)

@christineweng would you be ok if we fix this in a separate PR? I don't like to push code that we know isn't working, but that issue was opened 2 years ago, so I'm guessing fixing it might not be straightforward and would make this PR's scope a bit (lot?) bigger... Let me know!

Reply:
I think it's fine without this fix for now. There's a workaround for it, but the main issue here is really, should we essentially "auto-fill" the correct indices for the data requested if we know them even though it doesn't align with sourcerer? It's tough to say, because if we filter the events page now by an alert field from that flyout it won't show up...unless we also auto add the alerts index to sourcerer as well.. And then it would be weird if we disable alert fields on the events page. Not sure of a good solution....(maybe opening the alerts page with the flyout in a new tab when you click expand?)

Reply:
Yep, you do understand correctly @christineweng !